helmfile/charts/prometheus-alerts/files/disk-perf.yaml

```diff
@@ -0,0 +1,46 @@
+groups:
+  - name: disk-performance
+    rules:
+      - alert: DiskReadWaitTimeHigh
+        annotations:
+          description: Disk {{ $labels.device }} Wait Time on {{ $labels.instance }} is {{ $value }}, check the workload or adjust this alert
+          summary: Disk {{ $labels.device }} Wait Time is high on {{ $labels.instance }}
+          severity_level: warning
+          storage_type: local
+        expr: |
+          (
+          rate(node_disk_read_time_seconds_total{job="node-exporter"}[1m])
+          /
+          rate(node_disk_reads_completed_total{job="node-exporter"}[1m])
+          ) > 0.01
+        for: 30s
```

Comment:
this comments if for all the alerts.
How did you decide on the thresholds and the duration (for: 30s)? 
Looking at my dev cluster the `DiskQueueSizeHigh` would generate a lot of alerts that will open and close fast.

Reply:
ok, can you add the option to completely disable them from the overwrite configs?
it would be good to have this option in case they are too noisy

Reply:
Turned up to `for: 10m`.